sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcRelation.scala

```diff
@@ -194,6 +194,16 @@ private[sql] class OrcRelation(
   }
 
   override def prepareJobForWrite(job: Job): OutputWriterFactory = {
+    job.getConfiguration match {
```

Comment:
`OrcNewOutputFormat` also works in this case, but in general the current ORC data source is implemented upon the old `OrcOutputFormat` because we are still using Hive 0.13.1 right now. It would be confusing to mix `OrcNewOutputFormat` here.